### ISSUE TITLE
Evaluating on single frame task

### ISSUE THREAD

**Issue by davnords** [facebookresearch/vjepa2#95]
Hi I want to include V-JEPA 2 in my evaluation table for a single image task, how do I most fairly compute the embeddings for a single frame with this checkpoint?

```python
import torch
enc, _ = torch.hub.load('facebookresearch/vjepa2', 'vjepa2_vit_large')
```

When I simply run one frame through this I run into problems with the PatchEmbed3D that expects a time dimension.

/ David

**Comment by davnords** [comment]
I got an answer from the author, one should simply concat the image twice of the temporal axis and do the evaluation this way.

/ David
